js/modules/Debug/Debug.js

```diff
@@ -0,0 +1,984 @@
+            if (settings.type !== 'POST' && data === undefined) {
+                // get data from query string
+                data = {};
+                const query_string = settings.url.split('?')[1];
+                if (query_string !== undefined) {
+                    query_string.split('&').forEach((pair) => {
+                        const [key, value] = pair.split('=');
+                        data[key] = value;
+                    });
+                }
+            } else if (typeof data === 'string') {
+                // Post data is a URI encoded string similar to a query string. Values may be JSON strings
+                // so we need to parse it and convert it back to an object
+                const data_object = {};
+                data.split('&').forEach((pair) => {
+                    const [key, value] = pair.split('=');
+                    data_object[key] = decodeURIComponent(value);
+                    // try parsing the value as JSON
+                    try {
+                        data_object[key] = JSON.parse(data_object[key]);
+                    } catch (e) {
+                        // ignore
+                    }
+                });
+                data = data_object;
+            }
+            this.ajax_requests.push({
+                'id': ajax_id,
+                'status': '...',
+                'status_type': 'info',
+                'type': settings.type,
+                'data': data,
+                'url': settings.url,
+                'start': new Date(),
+            });
+            this.refreshWidgetButtons();
+        });
+
+        $(document).ajaxComplete((event, xhr, settings) => {
+            // If the request is going to the debug AJAX endpoint, don't do anything
+            if (settings.url.indexOf('ajax/debug.php') !== -1) {
+                return;
+            }
+            if (xhr.headers === undefined) {
+                return;
+            }
+            const ajax_id = xhr.headers['X-Glpi-Ajax-ID'];
+            if (ajax_id !== undefined) {
+                const ajax_request = this.ajax_requests.find((request) => request.id === ajax_id);
+                if (ajax_request !== undefined) {
+                    ajax_request.status = xhr.status;
+                    ajax_request.time = new Date() - ajax_request.start;
+                    ajax_request.status_type = xhr.status >= 200 && xhr.status < 300 ? 'success' : 'danger';
+                    // If the server sent a X-GLPI-Debug-Server-Global header, store it
+                    if (xhr.getResponseHeader('X-GLPI-Debug-Server-Global') !== null) {
+                        ajax_request.server_global = JSON.parse(xhr.getResponseHeader('X-GLPI-Debug-Server-Global'));
+                    }
+
+                    // Ask the server for the debug information it saved for this request
+                    $.ajax({
+                        url: CFG_GLPI['url_base'] + '/ajax/debug.php',
```

Comment:
```suggestion
                        url: CFG_GLPI.root_doc + '/ajax/debug.php',
```

Url base var should be used only for constructed one used outside of GLPI, like notifications